auto_dev/contracts/contract.py

```diff
@@ -38,6 +38,16 @@ def parse_functions(self) -> None:
             raise ValueError(msg)
         with abi_path.open("r", encoding=DEFAULT_ENCODING) as file_pointer:
             abi = json.load(file_pointer)["abi"]
+
+        # this is added because web is not compatible with pre-Solidity 0.6
+        for item in abi:
+            if item.get("type") == "function":
+                if "constant" in item:
+                    raise TypeError(
```

Comment:
If its just a name change, can we not use the exiting function to parse it rather than writing off a whole subset of a language?

Reply:
TBH, we could add an issue and fix this bug later